packages/web/src/pages/audio-rewards-page/ChallengeRewardsTile.tsx

```diff
@@ -353,8 +415,15 @@ const RewardsTile = ({ className }: RewardsTileProps) => {
         <LoadingSpinner className={wm(styles.loadingRewardsTile)} />
       ) : (
         <>
-          {isRewardsCooldownEnabled ? <ClaimAllPanel></ClaimAllPanel> : null}
-          <Divider orientation='horizontal' className={wm(styles.divider)} />
+          {isRewardsCooldownEnabled && !shouldHideCumulativeRewards ? (
+            <>
+              <ClaimAllPanel />
+              <Divider
+                orientation='horizontal'
+                className={wm(styles.divider)}
```

Comment:
do we need these divider styles? lmk if it's not working as expected and we can pair

Reply:
apparently we do need the class styles (otherwise nothing shows up). orientation is horizontal by default.